libpod/events/logfile.go

```diff
@@ -158,6 +158,10 @@ func rotateLog(logfile string, content string, limit uint64) (bool, error) {
 	}
 	file, err := os.Stat(logfile)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			// The logfile does not exist yet.
+			return false, nil
+		}
```

Comment:
I think this hides another problem. We didn't need this before and now it fails. That means something changed in c/common. I assume we do not get the default value want. I assume it is 0 for some reason.

Reply:
I don't think it's hiding a problem.  The default has changed from 0 to 1mb, so we did not hit the ErrNotExist case before.

Reply:
Why would it try to rotate a log which doesn't exists yet? This sounds very dangerous.

Reply:
I agree and now we don't rotate it anymore.

Reply:
Ok I understand it now, we always call rotateLog() before we even write an event. So of course the file cannot exists yet and we need to ignore this.